data.py

```diff
@@ -17,23 +17,28 @@ def statewide_by_day():
 
     return results
 
+
 def statewide_by_party_day():
     sql = """
         SELECT * FROM cumulative_stats_by_party_day
         WHERE days_before < 45 AND days_before >= (select '2021-01-05' - max(date(file_update_time - interval '18 hours')) from updated_times);
     """
 
     results = {}
-    results['combined'] = {}
+    results["combined"] = {}
     for result in db.engine.execute(text(sql)):
-        results.setdefault(result['party'] or '', {})[result['days_before']] = {
+        results.setdefault(result["party"] or "U", {})[result["days_before"]] = {
```

Comment:
is your goal that D + R + U = combined?

i think you might also want to include people that have "NP" as their party in your unaffiliated bucket.

Reply:
ah, good catch. I've updated the code so any designation other than D or R becomes U, and checked that the numbers add up now